src/server/api/schema.js

```diff
@@ -433,6 +437,32 @@ const rootMutations = {
 
       return updatedUser
     },
+    initiatePasswordReset: async (_, { organizationId, userId }, { user } ) => {
+      if (user.id !== userId) {
```

Comment:
What's the use-case here for logged-in users?  If they're logged in, presumably they know their password -- so maybe just simplify to just ADMINs?

Reply:
@schuyler1d if the current user is asking to initiate reset of their own password, we know they're already logged in, and no further authentication is necessary.  Otherwise, if a user is asking to initiate reset for someone else's password, we make sure the requestor is an admin.

Reply:
I understand the logic that's here -- I guess I'm wondering why expose the option for logged-in users for themselves at all?  If they can login, then they know their password, right?  Do you know any systems that have a "Reset Password" link from a logged-in state?

Reply:
What if somebody wants to change their password?  The way to change an auth0 password is to reset it.  That's how this came up -- an actual WFP user attempted to change his password when it only handled local auth.

Reply:
Ah, in that case, let's document that both in the code and the interface (for Auth0 case) -- they really don't have a change password prompt?

Reply:
Sure.  Document where?  One of the read-me files or in the code?

Reply:
In the code (i guess just above the if statement here)
-- i meant in the app for the user, but I guess this:
https://github.com/MoveOnOrg/Spoke/pull/1155/files#diff-eeddb37c19a7ca11ebfcbb65e85f46bdR74
basically covers it.